### PR TITLE
add github ISSUE_TEMPLATE files

### PR DESCRIPTION
**Pull request by cliffordfajardo**
## What is the change?
1. add `bug_report.md` to `bug-report.yml` to enable Github's form based issue template
   - https://youtu.be/qQE1BUkf2-s?t=23
2.  add `config.yml` file to help direct users to the helpful pages

## Motivation
- encourage's bug reporter's to put more care into their bug report before submission
- this may help maintainer's receive more detailed & higher quality bug report's
- adds helpful tips for user's during the process of creating a bug/issue report

## Demo of Change
this PR is similar to this one I created here for another repo recently 
- https://github.com/antvis/G6/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      value: |
+        Thank you for reporting an issue :pray:.
+
+        This issue tracker is for reporting bugs found in `remix-cookbook.com` (https://github.com/remix-cookbook/remix-cookbook.com).
+        If you have a question about how to achieve something and are struggling, please post a question
+        inside of `remix-cookbook.com` Discussions tab: https://github.com/remix-cookbook/remix-cookbook.com/discussions
+
+        Before submitting a new bug/issue, please check the links below to see if there is a solution or question posted there already:
+         - `remix-cookbook.com` Issues tab: https://github.com/remix-cookbook/remix-cookbook.com/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
+         - `remix-cookbook.com` closed issues tab: https://github.com/remix-cookbook/remix-cookbook.com/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed
```
Comment (vedovelli) [inline .github/ISSUE_TEMPLATE/bug_report.yml:15]: NIT: `Closed` instead of `closed`

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    validations:
+      required: true
+  - type: input
+    id: link
+    attributes:
+      label: Your Example Website or App
+      description: |
+        Which website or app were you using when the bug happened?
+        Note:
+        - Your bug will may get fixed much faster if we can run your code and it doesn't have dependencies other than the `remix-cookbook.com` npm package.
```
Comment (vedovelli) [inline .github/ISSUE_TEMPLATE/bug_report.yml:33]: This repo is not an NPM package. We should rephrase this line.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    about: Please ask and answer questions here.
+  - name: 💬 Remix Discord Channel
+    url: https://rmx.as/discord
+    about: Interact with other people using Remix 📀
+  - name: 💬 New Updates (Twitter)
+    url: https://twitter.com/remix_run
+    about: Stay up to date with Remix news on twitter
+  - name: 🍿 Remix YouTube Channel
+    url: https://rmx.as/youtube
+    about: Are you a techlead or wanting to learn more about Remix in depth? Checkout the Remix YouTube Channel
```
Comment (vedovelli) [inline .github/ISSUE_TEMPLATE/config.yml:14]: Shall we also include our Twitter account? https://twitter.com/remixcookbook

**Review by vedovelli — CHANGES_REQUESTED** [review]
Thanks for taking the time to open this PR! Much appreciated. Please check the comments.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (cliffordfajardo) [reply .github/ISSUE_TEMPLATE/config.yml:14]: Yes, good idea!💡

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (cliffordfajardo) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Updated

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (cliffordfajardo) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Updated - https://github.com/remix-cookbook/remix-cookbook.com/pull/26/commits/b47d25038fdf41c72856178ad90dc5fa1d1a84b8

**Pull request merged**